ckanext/cfpb_extrafields/templates/package/search.html

```diff
@@ -1,9 +1,10 @@
 {% ckan_extends %}
-{% block breadcrumb_content%}
+{% block breadcrumb %}
 {{ super() }}
-<span style="padding-left:450px;float:right;overflow:hidden">
+<span style="float:right;overflow:hidden">
```

Comment:
@mattboehm Ah, great that looks much better, didn't realize that's what happened.  Order is fine I think!